python/agents/application/cron.py

```
@@ -0,0 +1,24 @@
+from scheduler import Scheduler
+from scheduler.trigger import Monday
+
+
+class Scheduler:
+    def __init__(self) -> None:
+        self.trader = Trader()
+        self.schedule = Scheduler()
+
+    def start(self) -> None:
+        while True:
+            self.schedule.exec_jobs()
```

Pylint (3.3.7)

[error] Instance of 'Scheduler' has no 'exec_jobs' member

(E1101)